Makefile

```diff
@@ -76,11 +76,6 @@ migrate: .env
 	$(MANAGE) migrate
 
 
-.PHONY: types
```

Comment:
This is causing one of the checks to fail, should we just have the one check for pre-commit?

Reply:
Oh! I meant to mention in the PR, I could not for the life of me get mypy to work correctly so I stripped it out. It was failing on all sorts of weird issues in the settings.py file. I'll remove it from the workflows.

Reply:
Totally fine with that lol, mypy is good but can be a real PITA sometimes and for this particular PR it's too many moving parts.